english/money-money-money/green.md

```diff
@@ -0,0 +1,4 @@
+You walk into a room where there is money everywhere.
+Hundred dollar bills neatly stacked in stacks of [$1,000,000](https://www.google.com/search?q=%241,000,000+bill&espv=2&biw=1440&bih=799&source=lnms&tbm=isch&sa=X&ved=0CAYQ_AUoAWoVChMI1pqRrpjgyAIVjD4mCh2ZSQba#imgrc=mP3aNDX7DGB6UM%3A).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.